tests/example_podlist/bank_pod_list.json

```diff
@@ -0,0 +1,620 @@
+              },
+              {
+                "name": "default-token-hszmd",
+                "readOnly": true,
+                "mountPath": "/var/run/secrets/kubernetes.io/serviceaccount"
+              }
+            ],
+            "terminationMessagePath": "/dev/termination-log",
+            "terminationMessagePolicy": "File",
+            "imagePullPolicy": "Always",
+            "securityContext": {
+              "privileged": true
+            }
+          }
+        ],
+        "restartPolicy": "Always",
+        "terminationGracePeriodSeconds": 30,
+        "dnsPolicy": "ClusterFirst",
+        "serviceAccountName": "default",
+        "serviceAccount": "default",
+        "nodeName": "10.186.164.166",
+        "hostNetwork": true,
+        "hostPID": true,
+        "securityContext": {
+
+
+        },
+        "imagePullSecrets": [
+          {
+            "name": "default-secret"
+          },
+          {
+            "name": "default-secret-regional"
+          },
+          {
+            "name": "default-secret-international"
+          }
+        ],
+        "schedulerName": "default-scheduler",
+        "tolerations": [
+          {
+            "key": "node.alpha.kubernetes.io/notReady",
+            "operator": "Exists",
+            "effect": "NoExecute"
+          },
+          {
+            "key": "node.alpha.kubernetes.io/unreachable",
+            "operator": "Exists",
+            "effect": "NoExecute"
+          },
+          {
+            "key": "node.kubernetes.io/disk-pressure",
+            "operator": "Exists",
+            "effect": "NoSchedule"
+          },
+          {
+            "key": "node.kubernetes.io/memory-pressure",
+            "operator": "Exists",
+            "effect": "NoSchedule"
+          }
+        ]
+      },
+      "status": {
+        "phase": "Running",
+        "conditions": [
+          {
+            "type": "Initialized",
+            "status": "True",
+            "lastProbeTime": null,
+            "lastTransitionTime": "2018-03-25T11:24:27Z"
+          },
+          {
+            "type": "Ready",
+            "status": "True",
+            "lastProbeTime": null,
+            "lastTransitionTime": "2018-03-25T11:24:35Z"
+          },
+          {
+            "type": "PodScheduled",
+            "status": "True",
+            "lastProbeTime": null,
+            "lastTransitionTime": "2018-03-25T11:24:35Z"
+          }
+        ],
+        "hostIP": "10.186.164.166",
+        "podIP": "10.186.164.166",
+        "startTime": "2018-03-25T11:24:27Z",
+        "containerStatuses": [
+          {
+            "name": "cognetive",
+            "state": {
+              "running": {
+                "startedAt": "2018-03-25T11:24:34Z"
+              }
+            },
+            "lastState": {
+
+            },
+            "ready": true,
+            "restartCount": 0,
+            "image": "registry.ng.net/demo_k8s/cognetive:latest",
```

Comment:
I think `cognetive` is kind of IBM-internal. Maybe rename.

Reply:
done